specification/workloads/resource-manager/Microsoft.Workloads/connectors/preview/2023-10-01-preview/backupTypes.json

```diff
@@ -0,0 +1,849 @@
+          "description": "Tiering policy to automatically move RPs to another tier.\r\nKey is Target Tier, defined in RecoveryPointTierType enum.\r\nTiering policy specifies the criteria to move RP to the target tier.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TieringPolicy"
+          }
+        },
+        "snapshotBackupAdditionalDetails": {
+          "$ref": "#/definitions/SnapshotBackupAdditionalDetails",
+          "description": "Hana DB instance snapshot backup additional details."
+        }
+      }
+    },
+    "TieringPolicy": {
+      "description": "Tiering Policy for a target tier.\r\nIf the policy is not specified for a given target tier, service retains the existing configured tiering policy for that tier",
+      "type": "object",
+      "properties": {
+        "tieringMode": {
+          "description": "Tiering Mode to control automatic tiering of recovery points. Supported values are:\r\n1. TierRecommended: Tier all recovery points recommended to be tiered\r\n2. TierAfter: Tier all recovery points after a fixed period, as specified in duration + durationType below.\r\n3. DoNotTier: Do not tier any recovery points",
+          "enum": [
+            "Invalid",
+            "TierRecommended",
+            "TierAfter",
+            "DoNotTier"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "TieringMode",
+            "modelAsString": true
+          }
+        },
+        "duration": {
+          "format": "int32",
+          "description": "Number of days/weeks/months/years to retain backups in current tier before tiering.\r\nUsed only if TieringMode is set to TierAfter",
+          "type": "integer"
+        },
+        "durationType": {
+          "description": "Retention duration type: days/weeks/months/years\r\nUsed only if TieringMode is set to TierAfter",
+          "enum": [
+            "Invalid",
+            "Days",
+            "Weeks",
+            "Months",
+            "Years"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "RetentionDurationType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "UserAssignedIdentityProperties": {
+      "description": "User assigned managed identity properties.",
+      "type": "object",
+      "properties": {
+        "clientId": {
+          "type": "string"
+        },
+        "principalId": {
+          "type": "string"
+        }
+      }
+    },
+    "UserAssignedManagedIdentityDetails": {
+      "description": "User assigned managed identity details.",
+      "type": "object",
+      "properties": {
+        "identityArmId": {
+          "type": "string"
+        },
+        "identityName": {
+          "type": "string"
+        },
+        "userAssignedIdentityProperties": {
+          "$ref": "#/definitions/UserAssignedIdentityProperties"
+        }
+      }
+    },
```

Comment:
In other comment you called out the api restriction because of which you have had to do it like so. Please note the impact of this would be that customers when using the SDK might have to manually construct the object from the typical resource used across rest of the Azure.
Considering it as non-blocking for now.